### PR TITLE
Adjusted font sizes and text for mobile friendliness

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Used `this.game.config.(height/width)` instead of `options.window(Height/Width)` so the game can be made with smaller viewports and still have text centered. Good stuff.

Screenshots simulating a Pixel 2 XL:

<table>
<thead>
<th>Before</th>
<th>After</th>
</thead>
<tbody>
<tr>
<td>
Before screenshot with text overflowing
</td>
<td>
After screenshot with text fit within the screen
</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: src/scenes/start.js [hunk 1]
```diff
@@ -18,31 +18,53 @@ export default class StartScene extends Phaser.Scene {
     }
 
     create() {
-        this.start_player = this.physics.add.sprite(100, options.windowHeight - 100, "codey");
+        this.createTextElements();
```
Comment (JoshuaKGoldberg) [inline src/scenes/start.js]: Moving `this.createTextElements` to the beginning of `create()` meant Codey gets drawn on top of them.

**Comment by LinKCoding** [comment]
Not sure why it's looking different on Codecademy's versus local. Here's a screenshot of what I see locally:
image

**Comment by JoshuaKGoldberg** [comment]
@LinKCoding you might have an old version still? There should be two lines of big text.

**Comment by LinKCoding** [comment]
Ha... yea... I deleted the cache and it worked... 
On a different note, this new server seems to caching everything - so trying to test a branch is now a bit more tedious.

**Comment by LinKCoding** [comment]
The start and end screens look great, but the gameplay itself has Codey off screen (as well as score and pause screen)

**Pull request merged**